packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -0,0 +1,302 @@
+              providerId: provider.internalId,
+            };
+          }),
+        );
+        return containerConfigurations;
+      }),
+    );
+    return providerContainer.flat();
+  }),
+).then(value => (tmpProviderContainerConfiguration = value.flat()));
+
+$: providerContainerConfiguration = tmpProviderContainerConfiguration
+  .filter(configurationKey => configurationKey.value !== undefined)
+  .reduce((map, value) => {
+    const innerProviderContainerConfigurations = map.get(value.providerId) || [];
+    innerProviderContainerConfigurations.push(value);
+    map.set(value.providerId, innerProviderContainerConfigurations);
+    return map;
+  }, new Map<string, IProviderContainerConfigurationPropertyRecordedSchema[]>());
+
+async function startContainerProvider(
+  provider: ProviderInfo,
+  containerConnectionInfo: ProviderContainerConnectionInfo,
+): Promise<void> {
+  if (containerConnectionInfo.status === 'stopped') {
+    setContainerStatusIsChanging(provider, containerConnectionInfo);
+    await window.startProviderConnectionLifecycle(provider.internalId, containerConnectionInfo);
+  }
+}
+
+async function stopContainerProvider(
+  provider: ProviderInfo,
+  containerConnectionInfo: ProviderContainerConnectionInfo,
+): Promise<void> {
+  if (containerConnectionInfo.status === 'started') {
+    setContainerStatusIsChanging(provider, containerConnectionInfo);
+    await window.stopProviderConnectionLifecycle(provider.internalId, containerConnectionInfo);
+  }
+}
+
+async function restartContainerProvider(
+  provider: ProviderInfo,
+  containerConnectionInfo: ProviderContainerConnectionInfo,
+): Promise<void> {
+  await stopContainerProvider(provider, containerConnectionInfo);
+  await startContainerProvider(provider, containerConnectionInfo);
+}
+
+async function deleteContainerProvider(
+  provider: ProviderInfo,
+  containerConnectionInfo: ProviderContainerConnectionInfo,
+): Promise<void> {
+  if (containerConnectionInfo.status === 'stopped' || containerConnectionInfo.status === 'unknown') {
+    setContainerStatusIsChanging(provider, containerConnectionInfo);
+    await window.deleteProviderConnectionLifecycle(provider.internalId, containerConnectionInfo);
+  }
+}
+
+function setContainerStatusIsChanging(
+  provider: ProviderInfo,
+  containerConnectionInfo: ProviderContainerConnectionInfo,
+): void {
+  containerConnectionStatus.set(`${provider.name}-${containerConnectionInfo.name}`, {
+    inProgress: true,
+    status: containerConnectionInfo.status,
+  });
+  containerConnectionStatus = containerConnectionStatus;
+}
+</script>
+
+<div class="flex flex-1 flex-col p-2 bg-zinc-900">
+  <div>
+    <p class="capitalize text-xl">Resources</p>
+    <p class="text-sm text-gray-400">
+      Additional provider information is available under <a
+        href="/preferences/extensions"
+        class="text-gray-400 underline underline-offset-2">Extensions</a>
+    </p>
+  </div>
+  <div>
+    {#each $providerInfos as provider}
```

Comment:
what is displayed if there are no provider ?

Reply:
In this PR i didn't add anything. In the next one it would be like this https://design.penpot.app/#/view/c1a43240-198d-11ed-817f-f940e3befd53?page-id=7524576d-080c-80dd-8002-0e62d894c2d4&section=interactions&index=5&share-id=611a6365-8047-804f-8002-0e636fc64875 .
Do you want me to add that part here?

Reply:
I'm fine if it's coming just wanted to be sure that it'll come